shared/src/main/java/com/faforever/neroxis/mask/BooleanMask.java

```diff
@@ -749,12 +754,10 @@ public BooleanMask path(Vector2 start, Vector2 end, float maxStepSize, int numMi
                 while (location.getDistance(nextLoc) > maxStepSize && numSteps < size * size) {
                     List<Vector2> symmetryPoints = getSymmetryPoints(location, symmetryType);
                     if (inBounds(location) && symmetryPoints.stream().allMatch(this::inBounds)) {
-                        applyAtSymmetryPoints((int) location.getX(), (int) location.getY(), SymmetryType.TERRAIN,
-                                              (sx, sy) -> setPrimitive(sx, sy, true));
+                        setPrimitive((int) location.getX(), (int) location.getY(), true);
```

Comment:
I was finding issues with this `setPrimitive()` change. But not on all seeds.

If you use Seed: -5247121608802479157L with 5 teams and 5 spawns on mapsize 512.
Then the connections layer in the BasicTerrainGenerator does this:
![image](https://github.com/user-attachments/assets/829f96ac-14bd-4538-b53d-5528cbfab712)

I haven't dug much into this path() function but it seemed to need the applyAtSymmetryPoints() here.

Reply:
Yeah for some of the symmetry like diag there is a bit in getting the source points I still have to fix

Reply:
Hmm yeah, so my suggestion to use `applyAtSymmentyPoint()` does not work....
Not sure why but it somehow completely flattens the map....
![image](https://github.com/user-attachments/assets/705d46e5-5b08-4be1-84ab-3ab4dcb9119a)

So yeah, maybe ignore my suggestion.